specs/latest/2.0/index.html

```diff
@@ -2125,6 +2136,10 @@ <h4>Uniforms and attributes</h4>
           </span>
       </dt>
       <dd>
+        If either <code>program</code> or <code>location</code> were generated by a
+        different <code>WebGL2RenderingContext</code> than this one, generates
+        an <code>INVALID_OPERATION</code> error. <br><br>
```

Comment:
and returns null.

Reply:
Somewhat implicit due to the catch-all at the end regarding OpenGL errors, but thanks, I'll add it.